README.md

```diff
@@ -111,6 +111,10 @@ Simply add a dependency on the `support-v13` library alongside PermissionsDispat
 
 - [Migrating to 2.x](https://github.com/hotchemi/PermissionsDispatcher/wiki/Migrating-to-2.x)
 
+## ProGuard
+
+PermissionsDispatcher takes care of ProGuard. No extra setting is required.
```

Comment:
Good draft! Can you possibly fix like following? 🙇 
`PermissionsDispatcher bundles ProGuard setting in aar. No extra setting is required.`